src/main/java/ch/yth2021/charjar/discord/bot/ApplicationProperties.java

```diff
@@ -28,6 +28,10 @@ public String getDiscordClientId() {
         return secretProperties.getProperty("discord-client-id");
     }
 
+    public String getKickuserActivated() {
+        return Strings.isNullOrEmpty(secretProperties.getProperty("kick-users")) ? properties.getProperty("kick-users") : secretProperties.getProperty("kick-users");
```

Comment:
i think  you shouldn't look into secret properties for the kick-users property
secret properties should be used only for properties that are secrets